assignment4.txt

```diff
@@ -0,0 +1,11 @@
+Emma Ladouceur
+
+1. A character array in c separates each letter into its own section. s[10] = {"h", "e", etc..}
+whereas if you wanted it as a string you could simply put it as s[10] = {"hello"}. The cmputer still sees each space as 0-10, but it will recognize each section. 
+C doesn't have its own thing for a string, so in a sense a string is just a character array
```

Comment:
a string is basically a character array except it always includes a null terminator at the end ("\0").